### PR TITLE
(refonte) Front build system and home page 

### PR DESCRIPTION
**Pull request by gmougeolle**
Lots of WIP : 

- cache has been removed from home page
- search is not implemented
- stylemark and readme documentation missing for tabs behaviour
- "certified" badge is not working (or the dev fixtures do not include certified organizations)
- "données de référence" tab is missing, as well as "latest reuses" (collection is empty)
- language selector in the footer is not working
- some @@ for translations are probably missing

Apart from that, the home page works and display "real" information !

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -57,3 +57,6 @@ nosetests.xml
 .project
 .pydevproject
 .noseids
+
+udata.cfg
+stylemark/
```
Comment (abulte) [inline .gitignore:62]: Pourtant on a bien `insert_final_newline = true` dans le `.editorconfig` 🧐 😉

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ udata customizations for Etalab / Data.gouv.fr.
 
 ## Compatibility
 
-**udata-gouvfr** requires Python 2.7+ and [uData][].
+**udata-gouvfr** requires Python 2.7+ and [udata][].w
```
Comment (abulte) [inline README.md:9]:
```suggestion
**udata-gouvfr** requires Python 3.7 and [udata][].
```

**Review by abulte — COMMENTED** [review]
Nice! There's a lot of doc 😍 

I did not do a full review, and I probably won't be able to. I guess we'll just roll and see what happens 🤠.

Some general comments:
- I'm gonna be annoying but I think we need new lines at the end of files before merging
- do we want to commit the `_old` templates?
- about cache directives in the templates, is it not easier to migrate them now since we've a diff with the old templates?

File: .gitignore (hunk 1 above)
Comment (gmougeolle) [reply .gitignore]: Ah, le dév qui crée un `.editorconfig`, le remplit, puis ne l'applique pas. Remarque ça m'a permis de voir que certains anciens fichiers s'étaient incrustés lors du merge 🤓

**Comment by gmougeolle** [comment]
> I did not do a full review, and I probably won't be able to. I guess we'll just roll and see what happens 🤠.

🔥 🧯 

> Some general comments:
> 
> * I'm gonna be annoying but I think we need new lines at the end of files before merging

No problem, I applied the `.editorconfig` on the whole project and it should be better

> * do we want to commit the `_old` templates?

We can do without them but I find it nice to have the old reference without having to dig into the Git history or opening yet another GitHub tab. Some old templates contain data that is now used in other new templates, so it's pretty useful to keep them somewhere.
As I doubt that we'll have converted every existing template during this iteration, future developpers will probably be happy to find those files easily. Maybe we can commit them for now and remove them gradually when templates are totally done ?

> * about cache directives in the templates, is it not easier to migrate them now since we've a diff with the old templates?

I've put back the cache on `home.html` and `footer.html`. However I've added a new file (`header.html`) that is devoid of cache. Should we add cache on there too ?

File: udata_gouvfr/theme/templates/reuse/card.html [hunk 3]
```diff
@@ -1,57 +1,21 @@
-{% cache cache_duration, 'reuse-card', reuse.id|string, g.lang_code %}
```
Comment (abulte) [inline udata_gouvfr/theme/templates/reuse/card.html:1]: Put this one back too 🙏

**Comment by abulte** [comment]
> > * do we want to commit the `_old` templates?
> 
> We can do without them but I find it nice to have the old reference without having to dig into the Git history or opening yet another GitHub tab. Some old templates contain data that is now used in other new templates, so it's pretty useful to keep them somewhere.
> As I doubt that we'll have converted every existing template during this iteration, future developpers will probably be happy to find those files easily. Maybe we can commit them for now and remove them gradually when templates are totally done ?

Works for me!

> > * about cache directives in the templates, is it not easier to migrate them now since we've a diff with the old templates?
> 
> I've put back the cache on `home.html` and `footer.html`. However I've added a new file (`header.html`) that is devoid of cache. Should we add cache on there too ?

Okay for now, just put back the other one I found (cf review).

**Comment by gmougeolle** [comment]
Erratum : `header.html` is not new and did not contain a cache tag. I've commited the old version to keep it somewhere as the new one is not feature-complete yet (no banners for instance).

**Pull request merged**